Databases/VSSTester/DiskShadow/Invoke-CreateDiskShadowFile.ps1

```diff
@@ -244,45 +189,19 @@ function Invoke-CreateDiskShadowFile {
     $matchCondition = "^[a-z]:$"
     Write-Debug $matchCondition
 
-    if ($dbEdbVol -eq $dbLogVol) {
-        $nl
-        "Since the same volume is used for this database's EDB and logs, we only need a single drive"
-        "letter to expose the backup snapshot."
-        $nl
-
-        do {
-            Write-Host "Enter an unused drive letter with colon (e.g. X:) to expose the snapshot" -ForegroundColor Yellow -NoNewline
-            $script:dbSnapVol = Read-Host " "
-            if ($dbSnapVol -notmatch $matchCondition) {
-                Write-Host "Your input was not acceptable. Please use a single letter and colon, e.g. X:" -ForegroundColor red
-            }
-        } while ($dbSnapVol -notmatch $matchCondition)
+    if ($comp -eq 0) {
+        Write-Host "  Since the same volume is used for this database's EDB and logs, we only need a single drive"
+        Write-Host "  letter to expose the backup snapshot."
     } else {
-        $nl
-        "Since different volumes are used for this database's EDB and logs, we need two drive"
-        "letters to expose the backup snapshot."
-        $nl
-
-        do {
-            Write-Host "Enter an unused drive letter with colon (e.g. X:) to expose the DATABASE volume" -ForegroundColor Yellow -NoNewline
-            $script:dbSnapVol = Read-Host " "
-            if ($dbSnapVol -notmatch $matchCondition) {
-                Write-Host "Your input was not acceptable. Please use a single letter and colon, e.g. X:" -ForegroundColor red
-            }
-        } while ($dbSnapVol -notmatch $matchCondition)
-
-        do {
-            Write-Host "Enter an unused drive letter with colon (e.g. Y:) to expose the LOG volume" -ForegroundColor Yellow -NoNewline
-            $script:logSnapVol = Read-Host " "
-            if ($logSnapVol -notmatch $matchCondition) {
-                Write-Host "Your input was not acceptable. Please use a single letter and colon, e.g. Y:" -ForegroundColor red
-            }
-            if ($logSnapVol -eq $dbSnapVol) {
-                Write-Host "You must choose a different drive letter than the one chosen to expose the DATABASE volume." -ForegroundColor red
-            }
-        } while (($logSnapVol -notmatch $matchCondition) -or ($logSnapVol -eq $dbSnapVol))
+        Write-Host "  Since different volumes are used for this database's EDB and logs, we need two drive"
+        Write-Host "  letters to expose the backup snapshot."
+    }
 
-        $nl
+    $dbSnapVol = $DatabaseDriveLetter
+    if ($comp) {
+        $logSnapVol = $LogDriveLetter
+    } else {
+        $logSnapVol = $dbSnapVol
```

Comment:
Makes more sense to add this to the previous `if` statement.

Reply:
Done.

A huge portion of this file could use more improvement, but this is essentially the same logic from before this PR. We check `[string]::Compare()` for some reason instead of just doing -eq, and then we check whether they match once before writing the `add volume` lines, then we check again before writing the text about the drive letters, and then we check a third time before writing the `expose` lines.

Moving this into the previous if gets us down to 2 comparisons. Further improvement can be done in a future PR.